src/backend/marsha/core/tests/api/playlists/test_is_claimed.py

```diff
@@ -0,0 +1,101 @@
+"""Tests for the Playlist is-claimed API of the Marsha project."""
+
+from django.test import TestCase
+
+from marsha.core import factories
+from marsha.core.lti.user_association import clean_lti_user_id
+from marsha.core.models import ADMINISTRATOR, INSTRUCTOR
+from marsha.core.simple_jwt.factories import (
+    InstructorOrAdminLtiTokenFactory,
+    StudentLtiTokenFactory,
+)
+
+
+class PlaylistIsClaimedAPITest(TestCase):
+    """Test the API endpoint to check if a playlist is claimed."""
+
+    maxDiff = None
+
+    def test_playlist_is_claimed_anonymous(self):
+        """Anonymous users cannot check if a playlist is claimed."""
+        playlist = factories.PlaylistFactory()
+        response = self.client.get(
+            f"/api/playlists/{playlist.id}/is-claimed/",
+            content_type="application/json",
+        )
+
+        self.assertEqual(response.status_code, 401)
+
+    def test_playlist_is_claimed_student(self):
+        """Random logged-in users cannot check if a playlist is claimed."""
+        video = factories.VideoFactory()
+        jwt_token = StudentLtiTokenFactory(resource=video)
+
+        response = self.client.get(
+            f"/api/playlists/{video.playlist.id}/is-claimed/",
+            HTTP_AUTHORIZATION=f"Bearer {jwt_token}",
+            content_type="application/json",
+        )
+        self.assertEqual(response.status_code, 403)
+
+    def test_playlist_is_claimed_orga_administrator(self):
+        """Organization administrators can check if a playlist is claimed."""
```

Comment:
This is not an organization admin, but a LTI admin user. This is not the same thing